api/server/router/image/backend.go

```diff
@@ -31,9 +31,9 @@ type imageBackend interface {
 }
 
 type importExportBackend interface {
-	LoadImage(ctx context.Context, inTar io.ReadCloser, outStream io.Writer, quiet bool) error
-	ImportImage(ctx context.Context, ref reference.Named, platform *specs.Platform, msg string, layerReader io.Reader, changes []string) (dockerimage.ID, error)
-	ExportImage(ctx context.Context, names []string, outStream io.Writer) error
+	LoadImages(ctx context.Context, inTar io.ReadCloser, outStream io.Writer, quiet bool) error
```

Comment:
We _really_ need to deprecate that `quiet` option 😞. It should _never_ have been part of the API. I noticed that one some time ago (haven't checked exactly how it's used anywhere), but we can even consider to

- remove the option from the backend (instead making the API discard the output if it's set?)
- deprecate (gated by API version), and make sure the _client_ discards the output